src/Codeception/Module/SOAP.php

```diff
@@ -105,34 +124,34 @@ protected function onReconfigure()
 
     public function _depends()
     {
-        return ['Codeception\Lib\InnerBrowser' => $this->dependencyMessage];
+        return [\Codeception\Lib\InnerBrowser::class => $this->dependencyMessage];
     }
 
-    public function _inject(InnerBrowser $connectionModule)
+    public function _inject(InnerBrowser $connectionModule): void
     {
         $this->connectionModule = $connectionModule;
         if ($connectionModule instanceof Framework) {
             $this->isFunctional = true;
         }
     }
 
-    private function getClient()
+    private function getClient(): AbstractBrowser
     {
         if (!$this->client) {
-            throw new ModuleRequireException($this, "Connection client is not available.");
+            throw new ModuleRequireException($this, 'Connection client is not available.');
         }
         return $this->client;
     }
 
-    private function getXmlResponse()
+    private function getXmlResponse(): DOMDocument
     {
         if (!$this->xmlResponse) {
             throw new ModuleException($this, "No XML response, use `\$I->sendSoapRequest` to receive it");
         }
         return $this->xmlResponse;
     }
 
-    private function getXmlStructure()
+    private function getXmlStructure(): \Codeception\Util\XmlStructure
```

Comment:
FQDN here too

Reply:
In fact, I imagined you telling me that if it were simply called 'XmlStructure', the information that it belongs to \Codeception would be lost when reading this line.

But you're right. I'll change it.